modules/Template/PortalPublication.yaml

```diff
@@ -0,0 +1,60 @@
+#######################################
+# Portal          
+#######################################
+
+# Note: this template definition uses the alternate syntax with 'attribute'
+
+classes:
+  PublicationTemplate:
+    description: A generalized data model template for Publications to be hopefully used across dccs
+    notes:
+    - This structure is based on a CSV file from MC2 annotations with customized fields as attributes.
+    attributes:
+      grantNumber:
+        range: string
+        required: true
+      doi:
+        range: string
+        required: true
+      journal:
+        range: string
+        required: true
+      pmid:
+        range: string
+        required: true
+      title:
+        range: string
+        required: true
+      year:
+        range: integer
+        required: true
+      author:
+        range: string
+        multivalued: true
+        required: false
+      abstract:
+        range: string
+        required: true
+      accessibility:
+        range: string
+        required: true
+
+  PortalPublication:
+    is_a: PublicationTemplate
+    description: A slightly more specialized publication concept intended for the specific scope of the NF Portal.
+    attributes:
+      studyUse:
+        range: PublicationEnum
+        required: true
+      diseaseFocus:
+        range: string
```

Comment:
We do have a `DiseaseFocusEnum` (see PortalStudy for example), so we can stick it in the range here to reuse it